interop/client/client.go

```diff
@@ -183,6 +184,10 @@ func main() {
 			opts = append(opts, grpc.WithPerRPCCredentials(oauth.NewOauthAccess(interop.GetToken(*serviceAccountKeyFile, *oauthScope))))
 		}
 	}
+	if len(*serviceConfigJSON) > 0 {
+		opts = append(opts, grpc.WithDisableServiceConfig())
+		opts = append(opts, grpc.WithDefaultServiceConfig(*serviceConfigJSON))
```

Comment:
1-line:

```go
opts = append(opts, grpc.WithDisableServiceConfig(), grpc.WithDefaultServiceConfig(*serviceConfigJSON))
```

Reply:
done.

Thanks for the quick review. If this looks good, can you please merge? I don't have merge permissions on this repo (as noted in other PR)